scripts/ci/homebrew/generate-pypi-formula.sh

```
@@ -30,114 +30,93 @@ fi
+# Validate resources were generated
+RESOURCE_COUNT=$(echo "$RESOURCES" | grep -c "^  resource " || echo "0")
+if [[ "$RESOURCE_COUNT" -lt 5 ]]; then
+    log_error "Expected multiple resource stanzas but only found ${RESOURCE_COUNT}"
+    log_error "poet may have failed to analyze dependencies."
+    exit 1
+fi
+log_info "Generated ${RESOURCE_COUNT} resource stanzas from poet"
+
+# Create temp files for resources
+TMPDIR=$(mktemp -d)
+trap "rm -rf $TMPDIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)